src/Plugins/Devices/DeviceProxy_LibUSB.cpp

```diff
@@ -20,6 +20,7 @@ using namespace std;
 
 #define hex2(VALUE) setfill('0') << setw(2) << hex << (unsigned)VALUE << dec
 #define hex4(VALUE) setfill('0') << setw(4) << hex << VALUE << dec
+#define MAX_ATTEMPTS 5
```

Comment:
Well it's an experimental value, I also set it on 10 and nothing changes. I've tried to print the "attemps" value into the loop while using mass storages and I noticed that max attemps was always 2 or 3, so i set the threshold a bit higher. I've tried this code with a lot of flash drives and they all work good, but maybe it needs further testing.